[CBRD-24165] Remove unused function disk_auto_volume_expansion_daemon…

minjune8506 opened the pull request:
http://jira.cubrid.org/browse/CBRD-24165

**Purpose**
Remove unused function disk_auto_volume_expansion_daemon_init() and the related code.
They are not implemented so i removed them.

**Implementation**
removed functions
    - disk_auto_expand()
    - disk_auto_expansion_execute()
    - disk_auto_volume_expansion_daemon_init()
    - disk_auto_volume_expansion_daemon_destroy()

**Remarks**
N/A

minjune8506 committed:
[CBRD-24165] Remove unused function disk_auto_volume_expansion_daemon_init() and related codes (#3320)

http://jira.cubrid.org/projects/CBRD/issues/CBRD-24165

removed functions
    - disk_auto_expand()
    - disk_auto_expansion_execute()
    - disk_auto_volume_expansion_daemon_init()
    - disk_auto_volume_expansion_daemon_destroy()

wrlawodms commented on src/storage/disk_manager.c:
```diff
@@ -2452,72 +2445,6 @@ disk_volume_boot (THREAD_ENTRY * thread_p, VOLID volid, DB_VOLPURPOSE * purpose_
 
-#if defined (SERVER_MODE)
-int
-disk_auto_expand (THREAD_ENTRY * thread_p)
```
Please remove the declaration of it too.

minjune8506 replied:
there were no declarations at disk_auto_expand & disk_auto_expansion_execute

wrlawodms replied:
There is `disk_auto_expand ()` in storage/disk_manager.h.

minjune8506 replied:
@wrlawodms
Sorry I didn't see it.

minjune8506 reviewed (commented):
there were no declarations at disk_auto_expand & disk_auto_expansion_execute

Commit:
[CBRD-24165] Remove unused function disk_auto_volume_expansion_daemon_init() and related codes (#3336)

http://jira.cubrid.org/projects/CBRD/issues/CBRD-24165

remove declaration disk_auto_expand()

wrlawodms reviewed (approved):
LGTM. Thanks for your first contribution. 👍

minjune8506 commented:
can you assign me to this PR?

eido5 reviewed (approved):
It is nice modification.

hornetmj commented:
@mhoh3963 This is removing unused code. So it is OK when it passes the build test. Please merge it for us.
